tests/phpunit/payment-methods/class-wc-stripe-ocs-payment-gateway-test.php

```
@@ -258,6 +258,74 @@ public function provide_show_save_option_by_method_adaptive_pricing(): array {
+		$gateway->method( 'get_upe_enabled_at_checkout_payment_method_ids' )->willReturn( [ WC_Stripe_Payment_Methods::CARD ] );
+		$gateway->method( 'get_upe_enabled_payment_method_ids' )->willReturn(
+			[
+				WC_Stripe_Payment_Methods::CARD,
+				WC_Stripe_Payment_Methods::BANCONTACT,
+				WC_Stripe_Payment_Methods::IDEAL,
+			]
+		);
+
+		// The real constructor consumes get_upe_enabled_payment_method_ids(); run it
+		// only after the stub above is configured, or it reads null and fatals.
+		( new ReflectionClass( WC_Stripe_OCS_Payment_Gateway::class ) )->getConstructor()->invoke( $gateway );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '293', column '9'). (undefined)

(MissingImport)